src/docs/product/accounts/membership.mdx

```diff
@@ -16,7 +16,7 @@ Permissions in Sentry are broadly handled using [organization-level roles](#orga
 All users and their organization-level roles are listed in **Settings > Members**. Users can hold any of the following org-level roles:
 
 - **Billing** users can manage payment and compliance details.
-- **Org Members** can view most data in the organization and act on issues. They can join and leave teams, and access projects their teams own.
+- **Org Members** can view most data in the organization and act on issues. They can join and leave teams, and access projects their teams own. For Business and Enterprise plans, members can also create personal projects (if allowed by organization settings).
```

Comment:
This feature doesn't exits today. We should add "if allowed by organization settings" later when it's launched. 
@Dhrumil-Sentry do you want to add something around "this feature will be launched soon" or something